javascript/javascript1/week2/weather_wear.js

```diff
@@ -0,0 +1,21 @@
+function weatherBasedWear(temp){
+    if(temp<=0){
+        return `wear a long warm thermo/winter coats: its very cold`;
+    }else if(temp<=20){
+        return `you can wear normal dress with some jacket or sweater`;
+    }else if(temp<=35){
+        return `Wear half shorts and t-shirts`;
+    } else{
```

Comment:
Use consistent spacing.

Here you are using both `}else...` and `} else...` (notice the space between `}` and `else`.

Though both are technically correct I would recommend to use the one with a bit more "air". It makes reading the code a better experience.

Example:
```js
    if (temp<=0) {
        return `wear a long warm thermo/winter coats: its very cold`;
    } else if (temp<=20) {
        return `you can wear normal dress with some jacket or sweater`;
    } else if (temp<=35) {
        return `Wear half shorts and t-shirts`;
    } else {
        ...
```

But the most important thing is consistent style, both for your own sake but also for the entire development team.